client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/hooks/use-validate-verification-code.ts

```diff
@@ -53,7 +54,7 @@ export default function useValidateVerificationCode(): {
 				...( type === 'email' && {
 					// Replace if it exists, otherwise add it
 					emails: [
-						...oldContacts.emails.filter(
+						...( oldContacts.emails || [] ).filter(
```

Comment:
`oldContacts.emails` can be undefined when the API call to fetch the contacts fails, and the user adds an SMS contact.